### PR TITLE
Post style

### PR DESCRIPTION
**Pull request by henriquesdias**
Post desktop layout

### CODE REVIEW

**Review by Francksuel — APPROVED** [review]
Very good!

File: src/components/Post/Icon-delete.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+import { IoTrash } from "react-icons/io5";
+
+export default function IconDelete() {
+  return <IconDeleteStyle />;
+}
+const IconDeleteStyle = styled(IoTrash)`
+  color: white;
+  font-size: 19px;
+  margin-left: 9px;
+`;
```
Comment (SauloDriven) [inline src/components/Post/Icon-delete.js:11]: Niceeee

File: src/components/Post/icon-update.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { RiBallPenLine } from "react-icons/ri";
+import styled from "styled-components";
+
+export default function IconUpdate() {
+  return <UpdateStyle></UpdateStyle>;
```
Comment (SauloDriven) [inline src/components/Post/icon-update.js]:
```suggestion
  return <UpdateStyle/>;
```

**Review by SauloDriven — APPROVED** [review]
Nice work!

File: src/components/Post/Icon-delete.js (hunk 1 above)
Comment (henriquesdias) [reply src/components/Post/Icon-delete.js:11]: Thank you. I still need to do the responsive.

File: src/components/Post/icon-update.js (hunk 2 above)
Comment (henriquesdias) [reply src/components/Post/icon-update.js]: Good !

**Review by guilhermeicarofr — APPROVED** [review]
Nice!

**Pull request merged**